lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -187,7 +187,6 @@ module.exports = {
    */
   async checkLogGroupSubscriptionFilterResourceLimitExceeded() {
     const region = this.provider.getRegion();
-    const cloudWatchLogsSdk = new this.provider.sdk.CloudWatchLogs({ region });
```

Comment:
Ideally if this PR only covers only `request`util seclucion. Can we have changes as made in this file proposed with other PR ?

Reply:
Yes, no problem with moving that. Do we want to merge this other PR first or should I reintroduce the sdk member as part of the provider?

Reply:
I think it'll be better to first clean this up

Reply:
I've opened the new PR here: https://github.com/serverless/serverless/pull/8857. I will put this one as WIP while we close it (I need to at least fix the Bbpromises there).

Reply:
Great. Thank you!

Reply:
Rebased on master after closure of #8857 . Should be ok to work on it again :smile: